Makefile.pre.in

```diff
@@ -730,7 +730,8 @@ list-targets:
 
 .PHONY: build_all
 build_all:	check-clean-src check-app-store-compliance $(BUILDPYTHON) platform sharedmods \
-		gdbhooks Programs/_testembed scripts checksharedmods rundsymutil build-details.json
+		gdbhooks Programs/_testembed scripts checksharedmods rundsymutil \
+		$(shell [ -f pybuilddir.txt ] && cat pybuilddir.txt)/build-details.json
```

Comment:
Hmm nit question: I think that this is part of Makefile command at L940, It does not needed to be keep it as is?

Reply:
Ah, this is changed at this PR, got it :)